src/macrofier/engine.mjs

```diff
@@ -772,10 +795,14 @@ const generateImports = (json, templates) => {
     imports += getTemplate('/imports/x-method', templates)
   }
 
-  if (json['x-schemas'] && Object.keys(json['x-schemas']).length > 0) {
-    imports += Object.keys(json['x-schemas']).map(shared => getTemplate('/imports/default', templates).replace(/\$\{info.title\}/g, shared)).join('\n')
+  if (json['x-schemas'] && Object.keys(json['x-schemas']).length > 0 && !json.info['x-uri-titles']) {
```

Comment:
I took a look at the code in `microfiber/index.mjs` and this `x-uri-titles` is being written, but never used.

What was you goal in using it here? maybe we can just remove it altogether from index.mjs & engine.mjs.

Reply:
This was added to differentiate Common schema and module schema. I have found this is only available in common schema.